backend/s3/s3.go

```diff
@@ -907,6 +912,8 @@ const (
 	maxUploadCutoff     = fs.SizeSuffix(5 * 1024 * 1024 * 1024)
 	minSleep            = 10 * time.Millisecond // In case of error, start at 10ms sleep.
 
+	defaultURLExpireTime = fs.Duration(15 * time.Minute)
```

Comment:
From the `rclone link` docs

```
$ rclone link --help

rclone link will create or retrieve a public link to the given file or folder.

    rclone link remote:path/to/file
    rclone link remote:path/to/folder/

If successful, the last line of the output will contain the link. Exact
capabilities depend on the remote, but the link will always be created with
the least constraints – e.g. no expiry, no password protection, accessible
without account.
```

So this should probably be longer by default!

We should probably add an expiry parameter to the link command as quite a lot of the remotes support it... The other thing that some remotes support is deleting a public link.

Reply:
ok cool so I will add additional param